client/src/components/Header/HeaderCategory.tsx

```diff
@@ -46,7 +47,16 @@ const Category = styled.span`
   border-left: 1px solid lightgray;
 `;
 
-const CategoryListItem = styled.li`
+const CategoryListItem = styled.li<{ isSelected: boolean }>`
+  width: 6rem;
+  padding: 0.8rem 1.2rem;
+  cursor: pointer;
+  ${(props) => {
+    if (props.isSelected) return `background-color: rgba(200, 200, 200, 0.25);`;
+  }}
```

Comment:
```suggestion
  ${(props) => props.isSelected && `background-color: rgba(200, 200, 200, 0.25);`}
```

if문을 조금 짧게 줄일 수 있어요 ㅎㅎ

저희의 업보로 쌓일 수도 있지만... emotion/react의 css를 통해
```typescript
   css`background-color: rgba(200, 200, 200, 0.25);`
```
요로콤 표현하면 자동완성을 이용할 수도 있습니다!
물론 구현해야할 업보로 쌓이는...😅

Reply:
ㅋㅋㅋㅋ쿠ㅜㅜㅜㅜㅜㅜ css 함수 사용하면 걷잡을 수 없을 것같아 일부러 저렇게 했는데 들켜버렸군요...

Reply:
저도 리뷰 붙이고 나서 아차 싶어섴ㅋㅋㅋㅋㅋ
솔직히 안해도 될 거 같아요...😂